resources/monitoring/values.yaml

```diff
@@ -1968,3 +1968,6 @@ prometheus:
     ## https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#podmetricsendpoint
     ##
     # podMetricsEndpoints: []
+
+pushgateway:
```

Comment:
If the idea suggested above about checking for push gateway in `tenant-fetcher-job` is valid, I believe it should be a global field